_data/people.yml

```diff
@@ -201,7 +201,17 @@ freiburg:
         location_link: "https://goo.gl/maps/zEL3g3KVE8Q2"
         gitter: davelopez
         matrix: "@davel0pez:matrix.org"
-
+            
+    engynasr:
+        name: Engy Nasr
+        title: M.Sc. Inf., Researcher
+        website: http://research.bebatut.fr
+        email: nasr@informatik.uni-freiburg.de
+        location: "Build.: 079, Room: -1005"
+        location_link: "https://goo.gl/maps/zEL3g3KVE8Q2"
+        gitter: engynasr
```

Comment:
please also get an orcid (https://orcid.org) account :)

Have you been able to build this website locally?

Reply:
not yet, but I have the way from Helena " If you need to test locally it's a bit complicated, you need to setup a proxy that proxies both your galaxy + the GTN. There's a very short note at the end of https://docs.galaxyproject.org/en/latest/admin/special_topics/gtn.html "

 I will put it on my list to do it

Reply:
? No this is not needd, please read the README :) 

https://github.com/usegalaxy-eu/website#building

Reply:
Please try to build it locally and get familiar with it. We all add content regular to this website.

Thanks!

Reply:
I have successfully built it and currently familiarizing myself :) 

Thank u so much